src/metric_config.cpp

```diff
@@ -90,7 +98,7 @@ Status MetricConfig::parseMetricsArray(const rapidjson::Value& v) {
             }
         }
 
-        if (listSize == this->enabledFamiliesList.size()) {
+        if (this->enabledFamiliesList.size() <= listSize) {
```

Comment:
What is the logic behind this comparision?

When parsing metrics array we have empty `enabledFamiliesList`. This means `listSize` is always 0.